ProjectMelee/Source/ProjectMelee/Private/CP_HealthComponent.cpp

```diff
@@ -0,0 +1,48 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+#include "CP_HealthComponent.h"
+
+// Sets default values for this component's properties
+UCP_HealthComponent::UCP_HealthComponent()
+{
+	// Set this component to be initialized when the game starts, and to be ticked every frame.  You can turn these features
+	// off to improve performance if you don't need them.
+	PrimaryComponentTick.bCanEverTick = false;
```

Comment:
was wonderin wat this did

Reply:
As best as I can describe, `bCanEverTick` is used to have an object be considered every 'tick' of the system (I think every frame). It's useful for animation or colliders, things you want to check each frame for conditionals. In this case, the health isn't updated every frame, so I set this false.